src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/search/PersonMatchService.kt

```diff
@@ -41,7 +42,14 @@ class PersonMatchService(
     return@runBlocking highConfidencePersonRecords
   }
 
-  private fun getPersonRecords(personScores: List<PersonMatchScore>): List<PersonMatchResult> = personScores.mapNotNull {
+  fun examineIsClusterValid(cluster: PersonKeyEntity): IsClusterValidResponse = runBlocking {
+    checkClusterIsValid(cluster).fold(
+      onSuccess = { it },
+      onFailure = { throw it },
+    )
+  }
+
+  private fun getPersonRecords(personScores: List<PersonMatchScore>): List<PersonMatchResult> = personScores.sortedByDescending { it.candidateMatchProbability }.mapNotNull {
```

Comment:
This sort was removed in a previous PR, wasn't it, and moved higher up?
Is it moving back down to here now?